libraries/adsk/shaders/wood/parquet_3/parquet_03.mtlx

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+  <nodedef name="ND_parquet_03_shader" node="parquet_03" nodegroup="Shaders">>
+    <output name="surfaceshader" type="surfaceshader"/>
+  </nodedef>
+  <nodegraph name="NG_parquet_03_shader" nodedef="ND_parquet_03_shader">
+    <constant name="colorCorrect_diffuse" type="color3">
+      <parameter name="value" type="color3" value="1.12, 0.99, 0.63" />
+    </constant>
+    <extract name="extract_color3_1" type="float">
+      <input name="in" type="color3" nodename="remap_color3_1" />
+    </extract>
+    <multiply name="multiply_color" type="color3">
+      <input name="in1" type="color3" nodename="multiply_difuse" />
+      <input name="in2" type="color3" nodename="colorCorrect_diffuse" />
+    </multiply>
+    <multiply name="multiply_difuse" type="color3">
+      <input name="in1" type="color3" nodename="parquet_03_basecolor" />
+      <input name="in2" type="color3" value="1.1, 1.1, 1.1" />
+    </multiply>
+    <normalmap name="normalmap_1" type="vector3">
+      <input name="in" type="vector3" nodename="parquet_03_normal" />
+    </normalmap>
+    <image name="parquet_03_basecolor" type="color3">
+      <parameter name="file" type="filename" value="textures/parquet_03/2k/parquet_03_basecolor_2k_acescg.exr" />
```

Comment:
I think this is a suitable way to specify texture dependencies. i.e from root of library. We could do it other ways.
I have discussed resolution dependence a bit and ideally the textures can be bound at usage time instead of here but we'd need a way to define the mapping. Another possibility is to use tokens to specify the "resolution" folder. i.e. instead of "2k" it would be %RESOLUTION% and would be set dynamically. Still TBD.